Dockerfile

```diff
@@ -70,7 +76,10 @@ RUN set -ex && apk --update --no-cache add \
     libc6-compat \
     ca-certificates \
     nodejs \
-    nodejs-npm
+    nodejs-npm \
+    python \
+    py2-pip && \
+    pip install protobuf
```

Comment:
can you explain why the python packages are necessary?
does the plugin require them? because looking at the download page it seems like all the dependencies are included, no?

Additionally, I wonder if we should specify the version of this packages.